src/localization/locales/en-US.json

```diff
@@ -244,15 +244,28 @@
         "form_boundary_geojson_title": "Upload your landscape’s boundary",
         "form_location_placeholder": "Landscape location",
         "form_location_select": "Select location",
-        "form_name_placeholder": "Your landscape’s official name"
+        "form_name_placeholder": "Your landscape’s official name",
+        "boundaries_file_no_accepted": "No accepted files",
```

Comment:
issue: Need to better explain what the error is.

Reply:
@paulschreiber This is the message you added in the Boundaries dropzone when there are no accepted files, I just moved this to be translated, see: https://github.com/techmatters/terraso-web-client/pull/331/files#diff-378e736e2329b798aa6afb45a118911249c370f297dd595d9998bea24b1c1092L75 . What should be the text here?